src/main/java/seedu/duke/commands/SetCommand.java

```diff
@@ -23,13 +24,17 @@ public SetCommand(Template template, String component, double value) {
      */
     @Override
     public void execute(Ui ui) throws DukeException {
-        LoadComponent c = template.getComponent(component);
+        if (component.equals("v")) {
+            template.setInitialPowerSupply(value);
+            ui.printSetVoltageSource(template.getInitialPowerSupply());
```

Comment:
You could consider to just `break` here and let the rest of the function get carried out instead of having `else`.
```suggestion
            ui.printSetVoltageSource(template.getInitialPowerSupply());
            break;
```

Reply:
So, remove the else and `break;` in the if block? Will `break` work for a method to return to call.. or should I just empty `return;`?

Reply:
Ah ya that's what I meant.